CONTRIBUTING.md

```diff
@@ -1,6 +1,50 @@
-# Contributing to Materialize demos
+# Contributing to Materialize Connection Examples
 
-Thank you for your interest in Materialize demos! Contributions of many kinds
-are encouraged and most welcome.
+Thank you for your interest in Materialize connection examples!
+Contributions of many kinds are encouraged and most welcome.
+
+## Pull requests
+
+We welcome pull requests from everyone. By participating in this project, you
+agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Do we have a code of conduct somewhere?

Reply:
Ah yes, good catch. I've just removed this reference!